src/client/components/HomePage/StatisticsGraphic/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { createStyles, makeStyles } from '@material-ui/core'
+import statsGraphic from '~/assets/landing-page-graphics/stats-graphic.svg'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      maxWidth: '100%',
+    },
+  }),
+)
+
+const StatisticsGraphic = () => {
+  const classes = useStyles()
+
+  return (
+    <img className={classes.root} src={statsGraphic} alt="Statistics graphic" />
```

Comment:
In the `img` tag, can we add a `draggable="false"`, or a `userDrag="false"` in the jss to make the image not draggable?